bundle.Dockerfile

```diff
@@ -27,6 +27,8 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=operator-certification-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0
+LABEL operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
Are we sure this does the right thing? Can labels be specified as 'LABEL key: value'? The others are 'LABEL key=value'.

Reply:
reverted